build/bootstrap.proj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.80">
+    <PropertyGroup>
+        <RestorePackagesPath>$(MSBuildThisFileDirectory)../packages/</RestorePackagesPath>
+        <BaseIntermediateOutputPath>$(RestorePackagesPath)obj/</BaseIntermediateOutputPath>
+        <IntermediateOutputPath>$(BaseIntermediateOutputPath)$(Configuration)/</IntermediateOutputPath>
```

Comment:
Why do we need this?

Reply:
It's an attempt to move the generated obj folder outside of `$NUGET_ROOT\build\` folder

Reply:
This is a  no build SDK, so only NuGet is the one that creates assets, and NuGet reads MSBuildProjectExtensionsPath

Reply:
Ok, I will use `$(MSBuildThisFileDirectory)` instead of `$(BaseIntermediateOutputPath)`